spanner/tests/unit/test_pool.py

```diff
@@ -162,15 +162,16 @@ def test_bind(self):
     def test_get_non_expired(self):
         pool = self._make_one(size=4)
         database = _Database("name")
-        SESSIONS = [_Session(database)] * 4
+        SESSIONS = sorted([_Session(database) for i in range(0, 4)])
```

Comment:
Creating different session objects instead of copying the single one - needed for assertion